_specs/ecip-1079.md

```diff
@@ -0,0 +1,128 @@
+---
+lang: en
+ecip: 1079
+title: Phoenix EVM and Protocol Upgrades
+author: Bob Summerwill (@bobsummerwill)
+status: Draft
+type: Meta
+created: 2020-01-19
+license: Apache-2.0
+discussions-to: https://github.com/ethereumclassic/ECIPs/issues/263
+---
+
+### Simple Summary
+
+This ECIP describes a hard-fork which would replace
+[ECIP 1061: Aztlán EVM and Protocol Upgrades (Yingchun Edition)](https://ecips.ethereumclassic.org/ECIPs/ecip-1061) to "fix" two issues with ECIP-1061 where the written specification
+did not match the original intent.
```

Comment:
odd new line

Reply:
"Odd" referring to "not match the original intent", @BelfordZ?

The scope which we agreed to in the meeting was "Istanbul - 1884" with an implicit understanding that removing 1884 was removing the breaking gas-repricings.  When Wei came to do the implementation, he found that we had actually misspecified, because we had "thrown the baby out with the bathwater" in two ways - firstly with the SELFBALANCE opcode, and second in an interaction between two of the EIPs which assumed that 1884 was present.

Phoenix addresses those two mis-specifications.   All this detailed by Wei on his website.

Reply:
my bad, I meant purely formatting-wise. It's an odd choice to add a new line after "specification" and before "did"

Reply:
Right.  Not sure what my fingers and brain were thinking at the time :-)